website/src/word-panel.css.tsx

```diff
@@ -4,32 +4,76 @@ import sprinkles, {
   colors,
   hspace,
   mediaQueries,
+  radii,
   theme,
+  thickness,
   vspace,
 } from "./sprinkles.css"
 
+const wordPanelPadding = "8px"
+
 export const cherHeader = style({
   color: colors.headings,
   fontFamily: theme.fonts.cherokee,
 })
 
 export const wordPanelButton = styleVariants({
   basic: [closeButton],
+  colpright: [
+    closeButton,
+    {
+      position: "relative",
+      float: "right",
+      top: "0px",
+    },
+  ],
+  colpleft: [
+    closeButton,
+    {
+      position: "relative",
+      float: "left",
+      top: "0px",
+      marginRight: wordPanelPadding,
+      left: "0px",
+    },
+  ],
+})
+
+export const collPanelContent = style({
+  padding: wordPanelPadding,
+  marginBottom: "0px",
+})
+
+export const collPanelButton = style({
+  flex: 1,
+  backgroundColor: "rgba(0, 0, 0, 0)",
+  border: "none",
+  textAlign: "left",
+  padding: wordPanelPadding,
+})
+
+export const collPanel = style({
+  width: "inherit",
+  display: "flex",
+  flexDirection: "column",
+  border: "1px solid #ddd",
 })
 
 export const wordPanelContent = style({
   position: "sticky",
-  top: 125,
-  height: "calc(100vh - 150px)",
+  top: 100,
+
   border: "none",
   borderRadius: "4px",
-  padding: "8px",
+  padding: wordPanelPadding,
```

Comment:
Can we actually remove this padding so that the collapsible sections fill the width of the details panel? And then we can put the `h1` and `h2` inside of a `<header>` element which has this padding instead, to make sure those aren't up against the panel edge.

Reply:
Yeah! I didn't know about the header element before hand. I'll implement it.